content/actions/learn-github-actions/contexts.md

```diff
@@ -130,6 +130,7 @@ The `runner` context contains information about the runner that is executing the
 
 | Property name | Type | Description |
 |---------------|------|-------------|
+| `runner.name` | `string` | {% data reusables.actions.runner-name-description %} |
```

Comment:
Hey @fhammerl, thanks heaps for this PR! 🙇‍♂️ 

Normally, we don't create reusables for text that is only used in one place. The reusables for the other contexts in this table are there because those contexts are the same as similarly named environment variables that we document here: https://docs-10230--add-runner-name-de.herokuapp.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

I did a quick test, and it looks like the `runner.name` is available as the `$RUNNER_NAME` default environment variable.

Could you please add the environment variable to that article and use the same reusable for the description?

Reply:
Thanks, @lucascosti!

Of course, added the env var.

As a sidenote, should we mention that in the case of Hosted Runners, `runner.name` resolves to 'Hosted Runner' (or 'Hosted Runner {runnerIndex}' in case you have more than one in your pool)? 

Looking at the docs like they are, my first question would be 'So what's `runner.name` in case of Hosted Runners?'.

Reply:
Thanks, @fhammerl! Yep, I agree; feel free to add that to the description 👍

Reply:
Hmm, I went to add this change, but it looks like my tests the hosted runner name is "GitHub Actions <number>". I think we'll leave this for now, and follow-up later.